var/spack/repos/builtin/packages/arbor/package.py

```diff
@@ -6,68 +6,67 @@
 from spack import *
 
 
-class Arbor(CMakePackage):
+class Arbor(CMakePackage, CudaPackage):
     """Arbor is a high-performance library for computational neuroscience
     simulations."""
 
     homepage = "https://github.com/arbor-sim/arbor/"
-    url      = "https://github.com/arbor-sim/arbor/archive/v0.2.tar.gz"
+    git      = "https://github.com/arbor-sim/arbor.git"
+    maintainers = ['bcumming', 'halfflat']
 
-    version('0.4', sha256='7d9fc6b3262954cc5dc1751215fbb9f2cdb7010e829a2be43022f90da2d8d2e3')
-    version('0.2', sha256='43c9181c03be5f3c9820b2b50592d7b41344f37e1200980119ad347eb7bcf4eb')
+    version('master', branch='master')
+    version('0.5', commit="2158d2c61c0608363c66b9665ca4785e8c290b37", submodules=True)
 
-    variant('vectorize', default=False,
-            description='Enable vectorization of computational kernels')
-    variant('gpu', default=False, description='Enable GPU support')
+    variant('assertions', default=False, description='Enable arb_assert() assertions in code.')
+    variant('doc', default=False, description='Build documentation.')
     variant('mpi', default=False, description='Enable MPI support')
-    variant('python', default=False,
-            description='Enable Python frontend support')
-    variant('unwind', default=False,
-            description='Enable libunwind for pretty stack traces')
+    variant('neuroml', default=True, description='Build NeuroML support library.')
+    variant('python', default=True, description='Enable Python frontend support')
+    variant('vectorize', default=False, description='Enable vectorization of computational kernels')
 
-    depends_on('cuda', when='+gpu')
-    depends_on('mpi', when='+mpi')
-    depends_on('libunwind', when='+unwind')
+    # https://arbor.readthedocs.io/en/latest/install/build_install.html?highlight=requirements#compilers
+    conflicts('%gcc@:8.3')
+    conflicts('%clang@:8.0')
+    # Cray compiler v9.2 and later is Clang-based.
+    conflicts('%cce@:9.2')
```

Comment:
`:` is inclusive, so if you want to have < 9.2 conflict then do
```suggestion
    conflicts('%cce@:9.1')
```